### PR TITLE
[Wallet] Some minor cleanup

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

- Remove dead code related to Verifier app
- Move PGPNP flag to flags.ts
- Address some TODOs

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/locales/en-US/walletFlow5.json [hunk 1]
```diff
@@ -23,9 +23,6 @@
   "exchange": "Exchange",
   "viewAll": "View All",
   "thanksForVerifying": "Thanks for verifying",
-  "earnCeloRewards": "Earn Celo Rewards for verifying others",
```
Comment (aaronmgdr) [inline packages/mobile/locales/en-US/walletFlow5.json:26]: Nice

**Review by aaronmgdr — APPROVED** [review]

File: packages/mobile/src/account/reducer.ts [hunk 2]
```diff
@@ -61,7 +60,6 @@ export const initialState = {
   backupCompleted: false,
   backupDelayedTime: 0,
   socialBackupCompleted: false,
-  dismissedEarnRewards: false,
```
Comment (tarikbellamine) [inline packages/mobile/src/account/reducer.ts:64]: Should this also be removed from the `schema` file?

**Review by tarikbellamine — APPROVED** [review]
Minor comment about schema but otherwise looks good!

**Review by jeanregisser — APPROVED** [review]
Great 👍

File: packages/mobile/src/account/reducer.ts (hunk 2 above)
Comment (jmrossy) [reply packages/mobile/src/account/reducer.ts:64]: Good catch!

**Pull request merged**